webpack.config.js

```diff
@@ -65,11 +65,15 @@ const webpackConfig = {
 			{
 				include: /node_modules[\/\\]tinymce/,
 				loader: 'imports?this=>window',
+			},
+			{
+				test: /\.(ml|re)$/,
```

Comment:
Actually, I just checked with @rrdelaney and it looks like you don't need to explicitly list the interface files--bs-loader automatically respects them.